src/Components/Footer/footer.css

```diff
@@ -0,0 +1,6 @@
+/* TODO: Remove dummy styling */
+footer {
+  background-color: #f1f1f1;
+  padding: 1rem;
+  text-align: center;
+}
```

Comment:
This can all be removed, I just wanted to keep it so you could see the footer while working on the sticky footer